firmware/projects/TMS/inc/app.h

```diff
@@ -173,9 +173,15 @@ class BmsBroadcaster {
         bms_broadcast_.avg_therm_value = avg_thermistor_value;
         bms_broadcast_.num_therm_en = num_thermistors_;
         bms_broadcast_.therm_module_num = kThermistorModuleNumber;
+
         bms_broadcast_.checksum = CalculateBmsBroadcastChecksum(bms_broadcast_);
 
-        can_bus_.Send(bms_broadcast_);
+        auto checksum =             high_therm_id + high_therm_value +
```

Comment:
Remove the calculateBmsBroadcastChecksum function and just use the inline checksum calculation